src/subscription_manager/plugin/ostree/model.py

```diff
@@ -16,7 +16,10 @@
 import logging
 import re
 import os
-import subprocess
+
+# For python2.6 that doesn't have subprocess.check_output
+from rhsmlib.compat import check_output as compat_check_output
```

Comment:
Hehe. Went through all that effort of providing a compatible implementation, but then aliased? Can we just do `from rhsmlib.compat import check_output`? Then later (once we eventually drop 2.6), it just changes the import to `from subprocess import check_output`...